webcompat/form.py

```diff
@@ -147,6 +148,12 @@ def domain_name(url):
     # testing if it's an http URL
     if url.startswith(SCHEMES):
         domain = urlparse.urlparse(url).netloc
+        if domain.lower().startswith('www.'):
+            domain = domain.split('www.')[1]
+        port_number = re.search(r':\d+$', domain)
+        if port_number is not None:
```

Comment:
in Python it is not necessary to this construct. The `if` will execute only if the value is not None. see for example. so you can directly use the value.

```python
>>> import urlparse
>>> domain = 'http://www.org:5000'
>>> url = 'http://www.org:5000/foobar?blah'
>>> urlparse.urlparse(url)
ParseResult(scheme='http', netloc='www.org:5000', path='/foobar', params='', query='blah', fragment='')
>>> domain = urlparse.urlparse(url).netloc
>>> import re
>>> port_number = re.search(r':\d+$', domain)
>>> port_number
<_sre.SRE_Match object at 0x1097712a0>
>>> url2 = 'http://www.org/foobar?blah'
>>> domain2 = urlparse.urlparse(url2).netloc
>>> port_number2 = re.search(r':\d+$', domain2)
>>> port_number2
>>> if port_number:
...     print 'do something'
... 
do something
>>> if port_number2:
...     print 'do something'
... 
>>> 
```